spec/javascripts/packages/document-capture/context/service-provider-spec.js

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import ServiceProviderContext from '@18f/identity-document-capture/context/device';
+
+describe('document-capture/context/service-provider', () => {
+  it('defaults to undefined', () => {
+    const { current } = renderHook(() => useContext(ServiceProviderContext));
```

Comment:
I've been using `@testing-library/react-hooks` in some side-projects, and it seems like it's a nicer way to get at the context values. I think it could be used to replace a few ad hoc approaches we've used thus far:

https://github.com/18F/identity-idp/blob/9b3765367bd785083fa8aa18ed0b09475882d53b/spec/javascripts/packages/document-capture/context/device-spec.jsx#L6-L12

https://github.com/18F/identity-idp/blob/9b3765367bd785083fa8aa18ed0b09475882d53b/spec/javascripts/packages/document-capture/context/acuant-spec.jsx#L81-L93